WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -1089,10 +1088,11 @@ public void onClick(View v) {
                                     UploadUtils.publishPost(WPMainActivity.this, post, site, mDispatcher);
                                 }
                             });
-                    boolean isNewPost = data.getBooleanExtra(EditPostActivity.EXTRA_IS_NEW_POST, false);
-                    if (isNewPost && mBloggingRemindersManager.shouldShowBloggingRemindersPrompt(site.getId())) {
-                        mBloggingRemindersViewModel.showBottomSheet(site.getId(), Screen.PROLOGUE);
-                    }
```

Comment:
Since we're not using `mBloggingRemindersManager` here anymore, I believe we can remove its field declaration from this class, right?